test/test_service.js

```diff
@@ -25,29 +25,51 @@ describe('Global health check service', () => {
     });
 
     describe('When all services are responding correctly', () => {
-        it('should return http 200', (done) => {
+        let statusResults = [];
+        it('the service should return a success code 200 and an array of 3 items', (done) => {
             chai.request(svc.server).get('/status').end((err,res) => {
                 res.should.have.status(200);
-                //res.body.shoud.be.a('object'); //that.eql({something expected})
+                res.body.should.be.a('array');
+                res.body.length.should.eql(3);
+                statusResults = res.body;
                 done();
             });
         });
+        it('all 3 services should report a status = "OK"', () => {
+            statusResults[0].status.should.eql("OK");
+            statusResults[1].status.should.eql("OK");
+            statusResults[2].status.should.eql("OK");
+        });
+        it('all 3 services should report a detail = "OK"', () => {
+            statusResults[0].details.should.eql("OK");
+            statusResults[1].details.should.eql("OK");
+            statusResults[2].details.should.eql("OK");
+        });
     });
     describe('When one service does not respond', () => {
-        it('should return http 503', (done) => {
+        let statusResults = [];
+        it('the service should return an http error 503 and an array of 3 items', (done) => {
             //redefine mock urls to make one of them to fail:
             nock.cleanAll();
-            services.get('/auth/status').reply(500, "OK")
+            services.get('/auth/status').reply(500, "cannot reach server")
                     .get('/access/status').reply(200,"OK")
                     .get('/data/status').reply(200,"OK");
             chai.request(svc.server).get('/status').end((err,res) => {
                 res.should.have.status(503);
-                //res.body.shoud.be.a('object'); //that.eql({something expected})
+                statusResults = res.body;
+                res.body.should.be.a('array');
+                res.body.length.should.eql(3);
                 done();
             });
         });
+        it('should return an error for shoreline', () => {           
+            let authStatus = statusResults.filter(status => status.service == "shoreline");
+            authStatus[0].status.should.eql("NOK");
+            authStatus[0].error.should.eql("500 - \"cannot reach server\"");
+        })
     });
 
```

Comment:
would be great to add test cases for :
- return 200 but no body
- return 200 with malformed JSON body
- return 200 with JSON body
basically what was added in this PR

Reply:
Done ;)